### PR TITLE
Dashboard #398

### PR DESCRIPTION
**Pull request by wereinse**
## Type of PR

- [x ] Documentation changes
- [ ] Code changes
- [ ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

### Purpose of PR
To add commands to the README to allow a json dashboard to be imported for csharp, java or typescript.  
### Validation
- [ ] Unit tests updated and ran successfully
- [x] Update documentation or issue referenced above
 
### Issues Closed or Referenced
#398 
- Closes #<issue number> (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

File: testdash.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+nslookup ${He_Name}.azurewebsites.net
+nslookup ${He_Name}.vault.azure.net
+nslookup ${He_Name}.azurecr.ionslookup ${He_Name}.azurewebsites.net
+export He_Location=centralus
+export He_ACR_RG=${He_Name}-rg-acr
+export He_App_RG=${He_Name}-rg-app
+az group create -n $He_App_RG -l $He_Location
+az group create -n $He_ACR_RG -l $He_Location
+export Imdb_Name="imdbcosmoswsrdash2"
+az cosmosdb check-name-exists -n ${Imdb_Name}
```
Comment (aflinchb) [inline testdash.sh:16]: Should this be checked in?

File: README.md [hunk 2]
```diff
@@ -1,265 +1,287 @@
...
+# grant Key Vault access to each developer (optional)
+az keyvault set-policy -n $He_Name --secret-permissions get list --key-permissions get list --object-id $dev_Object_Id
+
+```
+
+Choose your app language
+
+- Choose which language version of the Helium container you want to build
+- Options are: [csharp](https://github.com/retaildevcrews/helium-csharp), [java (WIP)](https://github.com/retaildevcrews/helium-java), or [typescript (WIP)](https://github.com/retaildevcrews/helium-typescript)
+
```
[the remark this answers is not in the document]
Comment (wereinse) [reply README.md:169]: Yes

File: README.md [hunk 3]
```diff
@@ -1,265 +1,287 @@
...
+# run the tests in the container
+docker run -it --rm retaildevcrew/webvalidate --host https://${He_Name}.azurewebsites.net --files baseline.json
+
+```
+## Dashboard setup
+
+Replace the values in the `Helium_Dashboard.json` file surrounded by `%%` with the proper environment variables
+after making sure the proper environment variables are set
+
```
[the remark this answers is not in the document]
Comment (wereinse) [reply README.md:258]: Ok - He_Sub was added on line 45.  I've listed out (He_Sub, He_App_RG, Imdb_RB and Imdb_Name)

File: testdash.sh (hunk 1 above)
Comment (wereinse) [reply testdash.sh:16]: Nope - thank you

File: saveenv.sh [hunk 4]
```diff
@@ -28,3 +28,4 @@ else
   cat ~/${He_Name}.env
 fi
 
+export KEYVAULT_NAME="devshop-gelatodev-kv"
```
[the remark this answers is not in the document]
Comment (wereinse) [reply saveenv.sh]: Removed - thank you

File: docs/dashboard/Helium_Dashboard_variables.json [hunk 5]
```diff
@@ -0,0 +1,725 @@
...
+              "x": 0,
+              "y": 0,
+              "colSpan": 12,
+              "rowSpan": 1
+            },
+            "metadata": {
+              "inputs": [],
+              "type": "Extension/HubsExtension/PartType/MarkdownPart",
+              "settings": {
+                "content": {
```
[the remark this answers is not in the document]
Comment (wereinse) [reply docs/dashboard/Helium_Dashboard_variables.json:18]: When I create a dashboard and export the json - this is already there.  Not a great answer and I really don't know the answer to this one.

[the remark this answers is not in the document]
Comment (aflinchb) [reply docs/dashboard/Helium_Dashboard_variables.json:18]: Do we need the *_variables.json for import? If not, I think we should be ok not checking in

[the remark this answers is not in the document]
Comment (wereinse) [reply docs/dashboard/Helium_Dashboard_variables.json:18]: Not totally - but if the substitutions get made in our repo - we'd need it

File: testdash.sh [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+az cosmosdb check-name-exists -n ${He_Name}
+nslookup ${He_Name}.azurewebsites.net
+nslookup ${He_Name}.vault.azure.net
+nslookup ${He_Name}.azurecr.ionslookup ${He_Name}.azurewebsites.net
+export He_Location=centralus
+export He_ACR_RG=${He_Name}-rg-acr
+export He_App_RG=${He_Name}-rg-app
+az group create -n $He_App_RG -l $He_Location
+az group create -n $He_ACR_RG -l $He_Location
+export Imdb_Name="imdbcosmoswsrdash2"
```
[the remark this answers is not in the document]
Comment (wereinse) [reply testdash.sh]: yes removed testdash.sh

**Pull request merged**